tests/test_advanced_target_loop.py

```diff
@@ -0,0 +1,260 @@
+# This file is part of ts_scheduler
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+import os
+import glob
+import time
+import asyncio
+import logging
+import pathlib
+import unittest
+
+import numpy as np
+
+import lsst.ts.salobj as salobj
+
+from lsst.ts.scheduler import SchedulerCSC
+from lsst.ts.scheduler.utils.error_codes import NO_QUEUE
+from lsst.ts.scheduler.mock import ObservatoryStateMock
+
+with_scriptqueue = False
```

Comment:
This line probably is not needed since `with_scriptqueue` is always set in the try/except block two lines down.